src/Collection/ItemPriceCollection.php

```diff
@@ -218,6 +218,39 @@ public function resetDiscounts()
     }
 
     /**
+     * Merges this ItemPriceCollection with the given ItemPriceCollection to produce
+     * a new ItemPriceCollection for ItemPrices that share a key.
+     *
+     * The resulting ItemPriceCollection is composed of ItemPrices as constructed by
+     * the given comparator.
+     *
+     * Multiple items sharing the same key from the same collection are subject to
+     * being merged multiple times in the order in which they appear in the collection.
+     *
+     * @param ItemPriceCollection $collection The collection to be merged
+     * @param ItemComparatorInterface $comparator The
```

Comment:
`The`. The what? The square of the hypotenuse is equal to the square if its sides?